composer.json

```diff
@@ -23,8 +23,8 @@
         "sonata-project/block-bundle": "^3.0",
         "symfony-cmf/resource-rest-bundle": "^1.0@dev",
         "symfony-cmf/tree-browser-bundle": "^2.0@dev",
-        "symfony/framework-bundle": "^2.3 || ^3.0",
-        "symfony/property-access": "^2.2 || ^3.0"
+        "symfony/framework-bundle": "^2.8 || ^3.0",
+        "symfony/property-access": "^2.8 || ^3.0"
```

Comment:
Two reason:

* As @WouterJ said, Symfony CMF 2.0 is targetting Symfony 2.8+.
* The next major for other Sonata project will target SF 2.8 that is a LTS too: https://github.com/sonata-project/dev-kit/issues/51. So if we drop Symfony 2.3 right now, let's start on 2.8.